Dockerfile

```diff
@@ -4,8 +4,8 @@ WORKDIR /app
 
 COPY . .
 
-RUN pip install --no-cache-dir flask sqlalchemy psycopg2-binary
+RUN pip install --no-cache-dir flask sqlalchemy psycopg2-binary bcrypt

```

Comment:
You can create a requirements.txt file and install everything from there, ask ChatGPT so it'll give you an example.